### ISSUE TITLE
Проект заброшен?

### ISSUE THREAD

**Issue by lynrayy** [danyadev/vk-desktop#92]
Да, есть официальный вк мессенджер, но он ничем не лучше браузера, т.к это как и дискорд - тупо браузер.
Жрёт 500 мб ОЗУ, нет как таковых отличий от сайта, настроек вообще нет

Неужели никто не сделает (даже вы) десктоп клиент получше?

**Comment by danyadev** [comment]
Привет! Да, проект скорее заброшен, чем нет. Я изредка в отдельной ветке пишу новую версию, но такими темпами что-то юзабельное выйдет примерно никогда.

Однако мой проект на таком же электроне, что и дискорд и официальный вк мессенджер.
500мб звучит не так много, но это конечно зависит от доступной на машине оперативы...

В официальном мессенджере все-таки есть небольшие отличия от браузера, как и есть настройки

**Comment by lynrayy** [comment]
> 500мб звучит не так много

В 2007м году на диск ёмкостью 700мб влезала целая игра, а сейчас 500мб надо выделить в оперативке под какой-то мессенджер...
Вообще так глянул (да и в ответе ты написал), этот проект тоже на электроне написан, поэтому что там, что тут будет +- одинаково оперативы занято(

Ну окей. В любом случае, если тут будут какие-то необычные или популярные фишки, то это приложение было бы интереснее официального

**Comment by lynrayy** [comment]
В любом случае удачи, было бы неплохо, если бы проект возродился

**Comment by danyadev** [comment]
в последние годы игры могут не поместиться и в 50гб
для адекватной работы 11 винды уже нужны нехилые мощности
размер оперативы меньше 8гб, либо особенно меньше 4гб, считается слишком маленьким

такова реальность... на большие деньги 2007 года сегодня ты заживешь тоже далеко не так хорошо

**Comment by lynrayy** [comment]
Я к тому, что сейчас все обленились и юзают тяжёлые фреймворки вроде электрона хд
Понятное дело, самому писать всё долго и муторно, но у вк бесконечно денег, могли бы и вложиться...

**Comment by lynrayy** [comment]
Так, уже нашёл минусы официального мессенджера, хотелось бы это в кастомном видеть.

1. Только один аккаунт на всё приложение. Хотелось бы как в телеграме иметь два-три аккаунта парралельно и получать с них уведомления
2. Вк использует ущербные уведомления встроенные в винду. Ни звук настроить, ни сделать быстрые действия (их нет в виндусовских уведомлениях). Аналогично, хотелось бы иметь возможность включить кастомные как в телеграме: свой дизайн всплывающего уведомления, кнопки "Ответить" "Прочитано" и крестик, что бы закрыть всплывайку

**Comment by danyadev** [comment]
Потихоньку оживаем
